components/availability/src/Availability.svelte

```diff
@@ -1221,6 +1227,7 @@
   class:dated={allow_date_change}
   class:allow_booking
   on:mouseleave={() => endDrag(null, null)}
+  style="--free-color: {free_color}; --busy-color: {busy_color}; --partial-color: {partial_color};"
```

Comment:
@arjunkalburgi because we don't set the defaults in line 153-159, we would empty values here. Not sure if it causes any issues - but instead of using CSS for defaults - could we use the defaults set in 153-159?

Reply:
That change will keep it consistent with other variable defaults as well.

Reply:
yeah I tried it that way first actually, decided against it.

I did it this way because I wanted to props to overwrite the CSS vars. This is to stay consistent with the rest of the prop hierarchy. props are always highest hierarchy.  So if I add the default to L153-156, user will not be able to use CSS variables to modify the colours. 

The way it currently is works, because broken attributes just resolve themselves by disappearing!

If we don't want the ability for user to use CSS Vars for colours, then we can add the defaults to L153-156

Reply:
I wonder if css variables are right for this, rather than manifest props like anything else. We might get into some non-intuitive cascade situations: Manifest contains props, but user directly passed in --vars. I would expect, if it behaved like other manifest props, that the passed --vars would win.

Reply:
Right now we have both, Phil. I think there's a good case to rm external CSS variables though. Why support both?

Reply:
It's removed now